patches/minecraft/net/minecraft/world/item/UseAnim.java.patch

```diff
@@ -0,0 +1,20 @@
+--- a/net/minecraft/world/item/UseAnim.java
++++ b/net/minecraft/world/item/UseAnim.java
+@@ -1,6 +_,6 @@
+ package net.minecraft.world.item;
+ 
+-public enum UseAnim {
++public enum UseAnim implements net.minecraftforge.common.IExtensibleEnum {
+    NONE,
+    EAT,
+    DRINK,
+@@ -9,4 +_,9 @@
+    SPEAR,
+    CROSSBOW,
+    SPYGLASS;
++
++   public static UseAnim create(String name)
++   {
++       throw new IllegalStateException("Enum not extended");
++   }
```

Comment:
In Minecraft classes (patches), code should follow the formatting as defined in the rest of the class (by Fernflower): braces on newlines, and three spaces indentation.
```suggestion
+   public static UseAnim create(String name) {
+      throw new IllegalStateException("Enum not extended");
+   }
```